packages/web-pkg/src/components/FilesList/ResourceTile.vue

```diff
@@ -196,8 +196,10 @@ if (!lazy) {
 </script>
 
 <style lang="scss">
+.oc-tile-card.oc-card {
+  background-color: var(--oc-role-surface-container);
+}
```

Comment:
Nothing for now because it has always been like this, but having non-scoped and therefore global css instructions for the `oc-tile` class here feels a bit odd, we should improve that some day.

Reply:
There's more to finetune on the technical side 😅 At this point it might also make sense to introduce an OcCard component, which then has a color prop for defining the background-color. We sometimes need that (most of the time we don't), and when we need it it always gets hacky.